### ISSUE TITLE
About mask and time

### ISSUE THREAD

**Issue by hasayake007** [hiroaki-santo/deep-photometric-stereo-network#3]
Hello，I have some questions about this paper。。。。
1.  Each pixel of an image is treated as the training data in this paper. But I think there are lots of redundant data because of the existence of a mask. why ？
2. How long did it take to train this neural network?

Thank you！！

**Comment by hiroaki-santo** [comment]
Hi hasayake007,
1. We don't use the masked pixels for training. As you say, the Blobby dataset has lots of masked area and so, the actual number of the pixels used in training is smaller than 3.9e7.
2. I think it takes few hours or so.
